dts/bindings/ethernet/snps,ethernet-cyclonev.yaml

```diff
@@ -0,0 +1,17 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (C) 2022, Intel Corporation
+# Description:
+# Binding file for Ethernet driver CycloneV
+# heavily modified for Zephyr
```

Comment:
What is this for?

Also, please improve the commit message. `dts: arm:` should be something more descriptive, such as `dts: intel_socfgpa:` or so.